docs/source/configuring_deploying.rst

```diff
@@ -0,0 +1,298 @@
+
+
+Starting the Network
+--------------------
+
+Make sure you have all the Docker images you need:
+
+- sawtooth-validator (>= v1.1)
+- sawtooth-rest-api
+- sawtooth-intkey-tp-python
+- sawtooth-intkey-workload
+- sawtooth-settings-tp
+
+To build these from `sawtooth-core`_, run the following:
+
+.. _sawtooth-core: https://github.com/hyperledger/sawtooth-core
+.. code-block:: console
+
+  $ docker-compose -f docker-compose-installed.yaml build \
+  validator \
+  rest-api \
+  intkey-tp-python \
+  intkey-workload \
+  settings-tp
+
+
+Starting the Admin Service
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This compose file creates the shared network and volume required for the
+validator network to communicate.
+
+.. code-block:: console
+
+  $ docker-compose -f admin.yaml up -d
+
+
+Starting the Nodes
+~~~~~~~~~~~~~~~~~~
+
+If N is the number of nodes you want to create, startup N-1 nodes with the
+``node.yaml`` compose file. For each node, you will need to select a unique name
+and provide this with the ``-p`` option. Example:
+
+.. code-block:: console
+
+  $ docker-compose -p alpha -f node.yaml up
+
+Note that you can include the ``-d`` flag to create all nodes from a single
+terminal.
+
+For the last node, do the same as above, but also set the ``GENESIS`` environment
+variable. Example:
+
+.. code-block:: console
+
+  $ GENESIS=1 docker-compose -p genesis -f node.yaml up
+
+Note that each time a node is started, it adds itself to the list of nodes that
+all new nodes will connect to. This means that if you mess something up starting
+a node, you may need to start over. To do this, use ``docker-compose down`` for
+each node you started, passing the appropriate ``-p`` value; then do
+``docker-compose -f admin.yaml down``.
+
+
+Stopping and Restarting Nodes On a Network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To stop a node on a network, you need to stop the containers for that node
+**without removing them**. If you remove the containers, then that node is gone
+for good. To avoid for this, do **NOT** use ``docker-compose down``.
+
+If you are attached to the containers, you can press CTRL+C to stop the node's
+containers.
+
+If you are not attached to the containers, use the command ``docker-compose
+stop`` with the appropriate ``-p`` flag.
+
+After you have stopped a node, you can use ``docker-compose start`` with the
+appropriate ``-p`` flag to restart the stopped node.
+
+
+Applying Workload On a Network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A workload generator can be started against the network using the
+``workload.yaml`` compose file. To change the rate from the default, set the
+``RATE`` environment variable prior to running the compose command.
+
+For example, to start a workload of 2 TPS against the network, do:
+
+.. code-block:: console
+
+  $ RATE=2 docker-compose -f workload.yaml up
+
+To stop the workload, use ``docker-compose stop`` or ``docker-compose down``
+with ``workload.yaml``.
```

Comment:
Maybe easier to just pick one and use that command in its entirety i.e. 
```docker-compose -f workload.yaml down```
Ambiguity in directions tends to translate into questions on :rocket: chat.